BetaUtils.Core/Exceptions/Middleware/ResponseHeaderHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Net.Http.Headers;
+
+namespace BetaUtils.Core.Exceptions.Middleware;
+
+public static class ResponseHeaderHandler
+{
+    /// <summary>
+    /// Removing caching headers and setting status code of response.
+    /// </summary>
+    internal static void ClearResponse(HttpContext context, int statusCode)
```

Comment:
Yes there is. The class is left public so it could be (easier) tested. To test the method I've had to do reflection which was possible only because class access is set to public, if its been set to internal that would not be possible to test.

For more details check `ResponseHeaderHandlerTests.cs` Line: 66 😀